docs/variables.md

```diff
@@ -38,7 +42,9 @@ title: Variables References
 | Lib | `@{XDG_LIB_DIR}` | `.local/lib` |
 | GPG | `@{XDG_GPG_DIR}` | `.gnupg` |
 | SSH | `@{XDG_SSH_DIR}` | `.ssh` |
+| Private | `@{XDG_PRIVATE_DIR}` | `.{p,P}rivate {p,P}rivate` |
 | Passwords | `@{XDG_PASSWORD_STORE_DIR}` | `.password-store` |
+| Mail | `@{XDG_MAIL_DIR}` | `Mail .{m,M}ail` |
```

Comment:
Private and Mail are in two locations since it searches for both `directoryname` and `.directoryname`